submissions/denismaysov/multiPopup/index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MultiPopup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Oswald:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <div class="page__body">
+      <header class="header">
+        <div class="logo">
+          <a href="https://github.com/denismaysov" target="_blank">
+            <img src="img/user.svg" alt="User logo"
+          /></a>
+        </div>
+        <div class="search">
+          <input
+            placeholder="Search.."
+            class="search__input"
+            id="search-input"
+            type="search-input"
+          />
+          <label class="search__label" for="search-label">
+            <button class="search__button" type="submit" id="search-label">
+              Поиск
+            </button>
+          </label>
+        </div>
+      </header>
+      <div class="main">
+        <div class="main__row">
+          <nav class="option">
+            <div class="option__wrapper">
+              <ul class="option__menu">
+                <li class="option__menu-item">
+                  <label class="option__label" for="menu__popup">
+                    <img
+                      class="option__img"
+                      src="img/menu.svg"
+                      alt="Menu button"
```

Comment:
For decorative images implemented with `img`, the value of the `alt` attribute should be an empty value. Icon is a decorative image.

Reply:
its button which open popup, no?

Reply:
looks like that)